### PR TITLE
Discord domain verify (again xD)

### PR DESCRIPTION
**Pull request by xzonix**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel or Netlify.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.

### CODE REVIEW

File: _discord.fran.is-a.dev.json
Comment (21Z) [inline _discord.fran.is-a.dev.json:1]: Again, remove is-a.dev from the file name

**Review by 21Z — CHANGES_REQUESTED** [review]

File: _discord.fran.is-a.dev.json
Comment (CuteDog5695) [inline _discord.fran.is-a.dev.json:1]: File isn't in domains directory. (Also do what 21Z said)

**Review by iostpa — CHANGES_REQUESTED** [review]

**Comment by 21Z** [comment]
bruh

**Comment by 21Z** [comment]
Now create the _discord.fran.json with the txt record discord gave for fran.is-a.dev

**Comment by 21Z** [comment]
done in #14136, close this

**Pull request closed without merging**